tests/renderer/state-spec.ts

```diff
@@ -77,6 +76,10 @@ describe('AppState', () => {
     expect(appState).toBeTruthy();
   });
 
+  afterAll(async () => {
+    await new Promise((resolve) => setTimeout(resolve, 2000));
```

Comment:
This timeout waits for the `appState.runner` promise to resolve (most likely!) Otherwise CI throws an error saying -
```
ReferenceError: You are trying to `import` a file after the Jest environment has been torn down. From tests/renderer/state-spec.ts.
```
(As replicated [here](https://github.com/electron/fiddle/runs/7584809546?check_suite_focus=true#step:7:10))

I am quite baffled by this as I thought something like - `await appState.runner` will wait for the promise. Tried that but still CI throws an error.
After many rounds of trial and error, I came up with this _ugly_ solution. Let me know on how can this be resolved in a much better way(If not, then probably - I will add a comment to this code)